aws/outputs.tf

```diff
@@ -1,7 +1,3 @@
 output "bastion_ip" {
   value = "${aws_instance.bastion.public_ip}"
 }
-
-output "bosh_ip" {
-	value = "${aws_eip.bosh.public_ip}"
-}
```

Comment:
This static IP is required by the microbosh so that it can communicate with AWS in order to spin up new VMs, upload stemcells etc.